addons/mail/static/src/components/thread_viewer/thread_viewer.xml

```diff
@@ -4,12 +4,12 @@
     <t t-name="mail.ThreadViewer" owl="1">
         <div class="o_ThreadViewer" t-att-data-thread-local-id="threadViewer and threadViewer.thread and threadViewer.thread.localId">
             <t t-if="threadViewer">
-                <t t-if="threadViewer.isShowingLoading" name="loadingCondition">
+                <t t-if="threadViewer.isLoading" name="loadingCondition">
                     <div class="o_ThreadViewer_loading">
                         <span><i class="o_ThreadViewer_loadingIcon fa fa-spinner fa-spin" title="Loading..." role="img"/>Loading...</span>
                     </div>
                 </t>
-                <t t-elif="threadViewer.threadCache.isLoaded">
+                <t t-elif="threadViewer.threadCache.isLoaded or threadViewer.thread.isTemporary">
```

Comment:
It can be debated whether `isLoaded` means "no need to load again", or means literally "load rpc has been done", the name of the variable implying the latter, which is not the case for temporary thread.

Setting thread as "loaded" has (potentially) other implications than just this condition.
I don't think we should propagate the temporary notion further than necessary, and keep its reach on the rest of the code minimal.

Now if the variable was named "don't load messages again", then for sure I would set it `true` for temporary threads.